src/FSharpLint.Console/Program.fs

```diff
@@ -121,44 +124,64 @@ let private start (arguments:ParseResults<ToolArgs>) (toolsPath:Ionide.ProjInfo.
                exitCode <- ExitCode.Error
         | LintResult.Failure failure -> handleError ExitCode.Error failure.Description
     
-    let handleFixResult (ruleName: string) = function
+    let handleFixResult (ruleName: string) (checkFlag: bool) = function
         | LintResult.Success warnings ->
-            Resources.GetString "ConsoleApplyingSuggestedFixFile" |> output.WriteInfo
-            let increment = 1
+            if checkFlag |> not then
```

Comment:
@parhamsaremi no need for the use of |> operator here

Reply:
sure